colossus/src/main/scala/colossus/protocols/http/HttpBody.scala

```diff
@@ -5,7 +5,9 @@ import akka.util.ByteString
 import scala.util.Try
 import core.DataBlock
 
-class HttpBody(private val body: Array[Byte], val contentType : Option[HttpHeader] = None)  {
+class HttpBody(private val body: Array[Byte], contentTypeString: Option[String] = None) {
```

Comment:
We should not be making API changes in minor releases as it will break binary compatibility.  If we do this (which seems pretty reasonable to me), it'll have to go in the 0.10 release.